compiler/fir/entrypoint/src/org/jetbrains/kotlin/fir/checkers/CommonDeclarationCheckers.kt

```diff
@@ -58,6 +58,7 @@ object CommonDeclarationCheckers : DeclarationCheckers() {
         FirTypeParametersInObjectChecker,
         FirMemberFunctionChecker,
         FirMemberPropertyChecker,
+        FirOpenMemberChecker,
```

Comment:
`NON_FINAL_MEMBER_IN_OBJECT` indicates this diagnostic wants to visit and check `object`? Then, this should be in `classCheckers`, not `regularClassChecker`, which doesn't visit anonymous objects.

Reply:
Thanks! Fixed